pages/docs/tools/asyncapi-cli.md

```diff
@@ -0,0 +1,137 @@
+---
+title: AsyncAPI CLI
+weight: 1
+---
+
+## Install CLI Using NPM and Node
+To use the AsynAPI CLI tool, you must have Node.js version 10 or higher and NPM installed on your PC. To check if you already have both installed, run the following commands in your terminal:
+
+```sh
+# check if node is installed
+node -v
+# or
+node --version
+
+# check if NPM is installed
+npm -v
+# or
+npm --version
+```
+
+If you don’t have Node.js or NPM installed, [install both with this package manager](https://nodejs.org/en/download/package-manager/).
+
+After installing Node.js and NPM, run the following command to install the AsyncAPI CLI globally on your PC:
+```sh
+npm install -g @asyncapi/cli
+```
+
+## Install CLI on MacOS
+There are two ways to install the AsyncAPI CLI on your MacOS:
+
+### Using `brew`
+To install the AsyncAPI CLI using `brew` package manager, run the following commands in your terminal:
+```sh
+# Install brew
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+# Install AsyncAPI CLI
+brew install asyncapi
+```
+
+### Using `pkg`
+Every release of the AsyncAPI CLI has its own MacOS dedicated `pkg` file that enables you to install the CLI tool as a MacOS application. 
+To download the latest release of the CLI, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/latest/download/asyncapi.pkg
+```
+
+To download a specific release of the CLI, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/download/<replace this with the specific CLI version e.g v0.13.0>/asyncapi.pkg
+```
+
+All AsynAPI CLI releases are listed [here](https://github.com/asyncapi/cli/releases).
```

Comment:
```suggestion
<Remember>
Follow <a href="https://github.com/asyncapi/cli/releases">AsynAPI CLI releases</a>.
</Remember>
```
(1) This looks like perfect content for a `Remember` component, so adding one.

(2) For **accessibility**, you should never write anchor text that doesn't tell the user where a link will take them.

IOW, you should not write anchor text such as:
- read more
- here
- click here
- this
- etc.